api/python/api_key.md

```diff
@@ -12,14 +12,17 @@ grand_parent: API
 
 Sets an environment variable `"DC_API_KEY"` to given `api_key`.
 
-An API key is required to use the Python Client API. This can be provided to
+An API key is required to use the Python API. This can be provided to
```

Comment:
"An API key is required to use the Python API. This can be provided" -> "An API key can be provided"